libraries/eddystone-advertising-library/.jshintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "indent"  : 2,         // Number of spaces to use for indentation
+  "quotmark": "single",  // Require single quotes
```

Comment:
Style guide prefers single quotes:
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

I added a todo to include a linter in the tests and remove this option.